### PR TITLE
Feat/4413 publications in the notification wall

### PR DESCRIPTION
**Pull request by asf-algoo**
#4413

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [X] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: backend/tracim_backend/views/core_api/schemas.py [hunk 1]
```diff
@@ -1692,6 +1693,7 @@ class CommentSchema(marshmallow.Schema):
     parent_id = marshmallow.fields.Int(example=34, validate=positive_int_validator)
     content_type = StrippedString(example="html-document", validate=all_content_types_validator)
     parent_content_type = String(example="html-document", validate=all_content_types_validator)
+    parent_content_namespace = String(example="content", validate=all_content_namespaces_validator)
```
Comment (grignards) [inline backend/tracim_backend/views/core_api/schemas.py:1696]: Couldn't you have used `EnumField`?

File: frontend/src/container/NotificationWall.jsx [hunk 2]
```diff
@@ -110,56 +113,73 @@ export class NotificationWall extends React.Component {
...
+        : ''
+    )
 
     if (entityType === TLM_ENTITY.CONTENT) {
+      const publicationIcon = isPublication ? 'fas fa-stream+' : ''
+
       switch (eventType) {
         case TLM_EVENT.CREATED: {
           if (contentType === TLM_SUB.COMMENT) {
+            console.log("NOTIFICATIONCOMMENT", notification)
```
Comment (grignards) [inline frontend/src/container/NotificationWall.jsx:134]: Forgotten comment?

**Review by gbonaspetti — APPROVED** [review]

File: frontend/src/container/NotificationWall.jsx [hunk 3]
```diff
@@ -110,56 +113,73 @@ export class NotificationWall extends React.Component {
...
+        ? (
+          isPublication
+            ? PAGE.WORKSPACE.PUBLICATION(notification.workspace.id, notification.content.id)
+            : PAGE.WORKSPACE.CONTENT(notification.workspace.id, notification.content.type, notification.content.id)
+        )
+        : ''
+    )
 
     if (entityType === TLM_ENTITY.CONTENT) {
+      const publicationIcon = isPublication ? 'fas fa-stream+' : ''
```
Comment (grignards) [inline frontend/src/container/NotificationWall.jsx:129]: Is `fas` required here as it is always in the other part of the icon's class?

File: frontend/src/container/NotificationWall.jsx (hunk 2 above)
Comment (raphj) [reply frontend/src/container/NotificationWall.jsx:134]:
```suggestion
```

File: frontend/src/container/NotificationWall.jsx (hunk 3 above)
Comment (raphj) [reply frontend/src/container/NotificationWall.jsx:129]: it's actually two icons with the `+`, so nope, we need it there

**Review by Skylsmoi — APPROVED** [review]

File: backend/tracim_backend/views/core_api/schemas.py (hunk 1 above)
Comment (raphj) [reply backend/tracim_backend/views/core_api/schemas.py:1696]: done!

**Review by grignards — APPROVED** [review]

**Pull request merged**